### PR TITLE
Configure bor params in the unit itself

### PR DESCRIPTION
**Pull request by vcastellm**
## Motivation

This PR removes the necessity of having a startup script and an almost empty environment `metadata` file, less moving parts, and less files to watch.

## Implementation

As we are going to embed genesys data into the binary, this is the most similar method to do it before the new CLI release.

This way, each release gets tied to its corresponding genesys file.

This establish `bor` repository as the source of truth for the genesys file.

### CODE REVIEW

File: builder/files/bor.service [hunk 1]
```diff
@@ -6,11 +6,39 @@
...
+    --datadir /var/lib/bor/data \
+    --port 30303 \
+    --http --http.addr '0.0.0.0' \
+    --http.vhosts '*' \
+    --http.corsdomain '*' \
+    --http.port 8545 \
+    --ipcpath /var/lib/bor/bor.ipc \
+    --http.api 'eth,net,web3,txpool,bor' \
+    --syncmode 'full' \
+    --networkid '80001' \
```
Comment (ssandeep) [inline builder/files/bor.service:19]: This is for mumbai, how do we specify mainnet networkid 137?

File: builder/files/bor.service [hunk 2]
```diff
@@ -6,11 +6,39 @@
 [Service]
   Restart=on-failure
   RestartSec=5s
-  WorkingDirectory=$NODE_DIR
-  EnvironmentFile=/etc/matic/metadata
-  ExecStart=/usr/local/bin/bor $VALIDATOR_ADDRESS
+  ExecStart=/usr/local/bin/bor \
```
Comment (ssandeep) [inline builder/files/bor.service:9]: Can we make use of `--bor-mainnet` and `--bor-mumbai` instead?

File: .goreleaser.yml [hunk 3]
```diff
@@ -95,7 +101,9 @@ dockers:
       - linux-amd64
     build_flag_templates:
       - --platform=linux/amd64
-    skip_push: true
+    extra_files:
+      - builder/files/genesis-mainnet-v1.json
+      - builder/files/genesis-testnet-v4.json
```
Comment (ssandeep) [inline .goreleaser.yml:106]: If I want to run a devnet, I just need to add a new genesis file here and use that? How do I specify starting bor using the devnet genesis?

File: builder/files/bor.service (hunk 2 above)
Comment (vcastellm) [reply builder/files/bor.service:9]: I don't know, do they work correctly?

File: .goreleaser.yml (hunk 3 above)
Comment (vcastellm) [reply .goreleaser.yml:106]: The files are just there, you don't need to add devnet one here. The idea is that you mount a volume with your file and use that to init `bor` using you own genesys file.

File: builder/files/bor.service (hunk 1 above)
Comment (vcastellm) [reply builder/files/bor.service:19]: Gotcha, probably as you said it's better to use `--bor-mainnet` and `--bor-mumbai`

File: builder/files/bor.service (hunk 2 above)
Comment (ssandeep) [reply builder/files/bor.service:9]: Yes, it works perfectly. We have collected all the common params in there. You can compare it with those configs. Also, if there are additional configs or want to override any values, you can specify them after `--bor-mainnet`.

Comment (vcastellm) [reply builder/files/bor.service:9]: this is great, then I think we can set this to `--bor-mumbai` by default and leave a comment on `--bor-mainnet`

Comment (vcastellm) [reply builder/files/bor.service:9]: According to https://github.com/maticnetwork/bor/blob/7368ca6888c6c4850dce537d4aebf9229ba1a55b/cmd/geth/config.go#L342

the deafult sync mode is `snap`, should be `full` right?

Comment (ssandeep) [reply builder/files/bor.service:9]: Yes, it should be `full`.

**Review by ssandeep — APPROVED** [review]

File: builder/files/bor.service (hunk 2 above)
Comment (vcastellm) [reply builder/files/bor.service:9]: Changed in defaults

**Review by ferranbt — APPROVED** [review]

**Pull request merged**